draft-js-drag-n-drop-upload-plugin/src/handleDroppedFiles.js

```diff
@@ -26,84 +28,61 @@ export default function onDropFile(config) {
       // Set data {files: [Array of files], formData: FormData}
       const data = { files: [], formData };
       for (const key in files) { // eslint-disable-line no-restricted-syntax
-        if (files[key] && files[key] instanceof File) {
-          data.formData.append('files', files[key]);
-          data.files.push(files[key]);
+        const file = files[key];
```

Comment:
Getting a linting error here - `The body of a for-in should be wrapped in an if statement to filter unwanted properties from the prototype  guard-for-in` plz fix